lib/src/main/java/io/ably/lib/rest/Auth.java

```diff
@@ -43,7 +43,7 @@
 	 */
 	public enum AuthMethod {
 		basic,
-		token;
+		token
```

Comment:
Ideally I would use a comma here, for the benefit of future diffs.
```suggestion
		token,
```